ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ApprovedDeferrals.tsx

```diff
@@ -83,40 +87,30 @@ function ApprovedDeferrals() {
             ),
         [searchFilter, sortOption, page, perPage]
     );
-    const { data, loading, error } = useRestQuery(vulnerabilityExceptionsFn);
+    const { data, loading: isLoading, error } = useRestQuery(vulnerabilityExceptionsFn);
```

Comment:
Maybe early this release we can do a refactor so that this property is called `isLoading` from the return of `useRestQuery`?

Reply:
Sounds good. Here's the ticket for this one https://issues.redhat.com/browse/ROX-22865